pu_utils/aws/iam.py

```diff
@@ -146,6 +146,7 @@ def ses_policy(self, sender: str) -> RoleInlinePolicyArgs:
             policy=policy_json(
                 actions=[
                     "ses:SendEmail",
+                    "ses:SendRawEmail"
```

Comment:
Put a trailing slash here. This should be done by the formatter already.

```suggestion
                    "ses:SendRawEmail",
```